### ISSUE TITLE
Magento 2.2.6 Terms and Conditions are Not visible in Admin

### ISSUE THREAD

**Issue by Ctucker9233** [magento/magento2#18954]
### Preconditions (*)
Magento 2.2.6 fresh install

### Steps to reproduce (*)
1. Go to admin
2. Go to Store -> Terms and Conditions.
3. Add a New Condition
4. Save Condition

### Expected result (*)
1. New Condition should show up in the list of terms and conditions.

### Actual result (*)
1. No conditions, old or new, show up on the list.

**Comment by magento-engcom-team** [comment]
Hi @Ctucker9233. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@Ctucker9233 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by Ctucker9233** [comment]
@magento-engcom-team give me 2.2.6 instance

**Comment by magento-engcom-team** [comment]
Hi @Ctucker9233. Thank you for your request. I'm working on Magento 2.2.6 instance for you

**Comment by magento-engcom-team** [comment]
Hi @Ctucker9233, here is your [Magento instance](http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-2-6/).
Admin access: [http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-2-6//admin](http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-2-6//admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @Alex-Lunkov. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by oleksii-lunkov** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Alex-Lunkov. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Alex-Lunkov, here is your [Magento instance](http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-3-develop/).
Admin access: [http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-3-develop//admin](http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-3-develop//admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by oleksii-lunkov** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Alex-Lunkov. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Alex-Lunkov, here is your [Magento instance](http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-2-develop/).
Admin access: [http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-2-develop//admin](http://ec2-34-228-235-121.compute-1.amazonaws.com/i-18954-2-2-develop//admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by oleksii-lunkov** [comment]
The issue can't be reproduced on 2.3-develop and on 2.2-develop. Closing this issue.

**Comment by ghost** [comment]
working fine at  my end also.
![terms and conditions settings stores magento admin](https://user-images.githubusercontent.com/32056137/47741280-e6822600-dc9f-11e8-8fa5-b10de8f76fd2.png)

**Comment by ghost** [comment]
I was able to reproduce this issue. Issue comes when the prefix is added in table name. Reopening the issue.

**Comment by magento-engcom-team** [comment]
Hi @ayazwebkul. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
#squashtoberfest

**Comment by magento-engcom-team** [comment]
@ayazwebkul Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-96023` were created

**Comment by gwharton** [comment]
This is a duplicate of #18357 and has already had both 2.2 and 2.3 PR's produced.

Here is the 2.3 PR https://github.com/magento/magento2/pull/18412

Here is the 2.2 PR https://github.com/magento/magento2/pull/18866

The 2.2 change has yet to be merged.

**Comment by magento-engcom-team** [comment]
Hi @Ctucker9233. Thank you for your report. 
The issue has been fixed in magento/magento2#18866 by @gelanivishal in 2.2-develop branch
Related commit(s):
 - [2a6f8340de5fd6fd96c4d0848ba84e9e64cc1cf0](https://github.com/magento/magento2/commit/2a6f8340de5fd6fd96c4d0848ba84e9e64cc1cf0)

The fix will be available with the upcoming 2.2.9 release.
